{
  "public_data_at_usn": "2020-06-25T19:15:00Z",
  "crd": "0001-01-01T00:00:00Z",
  "candidate": "CVE-2020-10379",
  "public_date": "2020-06-25T19:15:00Z",
  "references": [
    {
      "reference": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10379"
    },
    {
      "reference": "https://github.com/python-pillow/Pillow/pull/4538"
    },
    {
      "reference": "https://github.com/python-pillow/Pillow/commits/master/src/libImaging"
    },
    {
      "reference": "https://pillow.readthedocs.io/en/stable/releasenotes/6.2.3.html"
    },
    {
      "reference": "https://pillow.readthedocs.io/en/stable/releasenotes/7.1.0.html"
    },
    {
      "reference": "https://ubuntu.com/security/notices/USN-4430-2"
    }
  ],
  "description": "In Pillow before 7.1.0, there are two Buffer Overflows in libImaging/TiffDecode.c.",
  "ubuntu_description": "",
  "notes": [],
  "bugs": [],
  "priority": "medium",
  "discovered_by": "",
  "assigned_to": "mdeslaur",
  "patches": [
    {
      "package_name": "pillow",
      "release_patches": [
        {
          "release_name": "focal",
          "status": "released",
          "note": "7.0.0-4ubuntu0.1"
        }
      ]
    },
    {
      "package_name": "pillow-python2",
      "release_patches": [
        {
          "release_name": "focal",
          "status": "needs-triage",
          "note": ""
        }
      ]
    },
    {
      "package_name": "python-imaging",
      "release_patches": [
        {
          "release_name": "focal",
          "status": "DNE",
          "note": ""
        }
      ]
    }
  ],
  "upstreams": [
    {
      "package_name": "pillow",
      "upstream_links": [
        {
          "link": "https://github.com/python-pillow/Pillow/pull/4507"
        }
      ]
    }
  ]
}
